### PR TITLE
Remove work fascinating from ToS.

### PR DESCRIPTION
**Pull request by AnnaMag**
#### Changes proposed in this Pull Request
Fixes https://github.com/Automattic/wp-calypso/issues/27234
and other uses of "fascinating".

#### Testing instructions
Changes to the wording are cosmetic.
To test e.g. the checkout location, initiate a plan upgrade process and verify the checkout screen copy
e.g. ```http://calypso.localhost:3000/checkout/$site/professional-monthly```

*before
screenshot 2019-01-09 at 12 35 17

*after
screenshot 2019-01-09 at 14 53 09

Note: PR includes lint fixes to
screenshot 2019-01-09 at 15 01 03

### CODE REVIEW

**Comment by AnnaMag** [comment]
Not adding Copy/Design review requests, because the change has been agreed on (e.g. #27234).

File: client/blocks/signup-form/index.jsx [hunk 1]
```diff
@@ -584,7 +583,7 @@ class SignupForm extends Component {
 			/>
 		);
 		const tosText = this.props.translate(
-			'By creating an account you agree to our {{a}}fascinating Terms of Service{{/a}}.',
+			'By creating an account you agree to our {{a}}Terms of Service{{/a}}.',
```
Comment (a8ci18n) [inline client/blocks/signup-form/index.jsx]: :ok: This change is safe, we'll keep the existing 26 translations.

File: client/jetpack-connect/site-url-input.jsx [hunk 2]
```diff
@@ -87,7 +87,7 @@ class JetpackConnectSiteUrlInput extends PureComponent {
 		return (
 			<p className="jetpack-connect__tos-link">
 				{ this.props.translate(
-					'By setting up Jetpack you agree to our fascinating {{tosLinkText}}Terms of Service{{/tosLinkText}} ' +
+					'By setting up Jetpack you agree to our {{tosLinkText}}Terms of Service{{/tosLinkText}} ' +
```
Comment (a8ci18n) [inline client/jetpack-connect/site-url-input.jsx]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'By setting up Jetpack you agree to our fascinating {{tosLinkText}}Terms of Service{{/tosLinkText}} and to sync {{syncLinkText}}certain data and settings{{/syncLinkText}} to WordPress.com' )` *ES Score: 26*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/checkout/checkout/terms-of-service.jsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ class TermsOfService extends React.Component {
 
 	renderTerms = () => {
 		let message = this.props.translate(
-			'By checking out, you agree to our {{link}}fascinating terms and conditions{{/link}}.',
+			'By checking out, you agree to our {{link}}terms and conditions{{/link}}.',
```
Comment (a8ci18n) [inline client/my-sites/checkout/checkout/terms-of-service.jsx]: :ok: This change is safe, we'll keep the existing 26 translations.

**Pull request merged**